docs/db/migrations.md

```diff
@@ -191,7 +191,7 @@ type Note {
   id: ID!
   title: String!
   """
-  @db.default: false
```

Comment:
Migrations are only for Postgres - since default values work for any data source there should also be a section added to [Data Model](https://graphback.dev/docs/intro/datamodel)

Reply:
Thanks for the pointers.